README.md

```diff
@@ -1,66 +1,120 @@
-kcsrv
+kcsrv (aka Kan'tColle Server)
 =====
-
-**kcsrv** is a project to try to implement an open source KanColle server.  
-It's currently under very heavy development. Check back later.
+**kcsrv** is a project to try to implement an open source KanColle server using Python 3.x.  
+It's currently under very heavy development. Check back later, but if you're really curious, read on and get yourself a copy.
 
 What Currently Works
 --------------------
-- Actually logging in (this is an achievement because of how temperamental KanColle is when it doesn't get the exact response it works back
+- Game login
+- First ship enlistment (Fubuki and friends)
 - Main port screen
 - Resources things (furniture, sound, etc)
 
+What Doesn't Work
+--------------------
+- Anything not listed above.
 
 FAQ
 ---
 
-### Q: Will I be able to import my data from an official server?
-
-Theoretically, yes. The only downside is it will require your API key to import your ships2 data and your admiral information. This may get your account banned.
-
-### Q: Why is it built on Python?
+### Q: Will I be able to import my data from an official DMM server?
+Theoretically, yes.   
+The only downside is it will require your API key to import your ships2 data and your admiral information. This may get your account banned for whatever lame DMM reason.  
+It is recommended you start fresh on your copy of this server. 
 
-There aren't actually that many things you can build a web application on, and most of them were out from the start.
+### Q: What is the recommended requirements?
+- Debian or Ubuntu based distro. Fedora and friends may work, but please, YMMV.
+- Python 3.x and knowledge about virtualenv (setup and activate).
+- Internet connection to download the DMM assets.
 
-* PHP is [an ungodly clusterfuck](http://eev.ee/blog/2012/04/09/php-a-fractal-of-bad-design/) that's insanely unwieldly, performs about as well as a football team in Antarctica and is a nightmare to maintain.  
-  I've written large applications in PHP before, and I'm not going there again.
+Protip: Use a VM (VMWare, VirtualBox). If you mess up, simply hose it and start squeaky clean. Or you can use an ARM based linux box (R-Pi, ODROID, etc).
 
-* Ruby and Java are both massive resource hogs, and this server needs to be able to scale well, preferably without me having me hire another server I can't pay for.  
-  I run a JIRA (Java) and a Discourse (Ruby) instance, and they both chew up nearly everything their servers have to give.
+### Q: Why is it built on Python?
+There aren't actually that many things you can build a web application on, and most of them were out from the start.    
+There was a large rant by one the kcsrv owner, I (SoftwareGuy) who improved this FAQ have condensed it down to make it less ranty.
 
+* PHP is [an ungodly clusterfuck](http://eev.ee/blog/2012/04/09/php-a-fractal-of-bad-design/).
+* Ruby / Java are both massive resource hogs, and this server needs to be able to scale well.  
```

Comment:
"Ruby and Java", the slash implies that they are in some way interchangeable, which they're definitely not. One is actually good (if you can afford the server for it).